macro/macro.go

```diff
@@ -18,6 +18,7 @@ type Macro interface {
 }
 
 func NewMacro(data string) (Macro, error) {
+	// TODO(jcchavezs): shall we fail if data is empty?
```

Comment:
I think so, invoking a macro with no data is mainly a programming mistake? e.g. using passing a `msg` with no text.